lib/workers/repository/update/branch/get-updated.spec.ts

```diff
@@ -28,6 +30,7 @@ jest.mock('../../../../modules/manager/composer');
 jest.mock('../../../../modules/manager/helmv3');
 jest.mock('../../../../modules/manager/npm');
 jest.mock('../../../../modules/manager/git-submodules');
+jest.mock('../../../../modules/manager/gomod');
```

Comment:
can't we simply use one of the existing managers? they are all mocked anyways 


```suggestion
jest.mock('../../../../modules/manager/gomod', () => mockDeep ());
```
perf, we should do it for all mocks